src/http/routing.rs

```diff
@@ -80,412 +145,534 @@ macro_rules! routes {
                 }))
             }
 
+            /// This returns the common identifiers, which are used in rate limit buckets.
+            /// If you are interested in less common identifiers,
+            /// please match the variants you are interested in.
```

Comment:
This documentation just duplicates the `RatelimitCause` docs, remove it.

Reply:
Removed